urbackupserver/www2/src/features/backups/BackupsTable.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  DataGrid,
+  DataGridHeader,
+  DataGridRow,
+  DataGridHeaderCell,
+  DataGridBody,
+  DataGridCell,
+  TableCellLayout,
+  TableColumnDefinition,
+  createTableColumn,
+} from "@fluentui/react-components";
+import { Link } from "react-router-dom";
+import { useSuspenseQuery } from "@tanstack/react-query";
+
+import type { BackupsClient } from "../../api/urbackupserver";
+import { getCellFocusMode } from "../../utils/table";
+import { formatDatetime } from "../../utils/format";
+import { urbackupServer } from "../../App";
+import { TableWrapper } from "../../components/TableWrapper";
+
+export const columns: TableColumnDefinition<BackupsClient>[] = [
+  createTableColumn<BackupsClient>({
+    columnId: "name",
+    renderHeaderCell: () => {
+      return "Computer name";
+    },
+    renderCell: (item) => {
+      return <TableCellLayout>{item.name}</TableCellLayout>;
+    },
+  }),
+  createTableColumn<BackupsClient>({
+    columnId: "lastFilebackup",
+    renderHeaderCell: () => {
+      return "Last file backup";
+    },
+    renderCell: (item) => {
+      return (
+        <TableCellLayout>
+          {item.lastbackup ? formatDatetime(item.lastbackup) : "-"}
+        </TableCellLayout>
+      );
+    },
+  }),
+];
+
+const REFETCH_INTERVAL = 5000;
+
+export function BackupsTable() {
+  const backupClientsResult = useSuspenseQuery({
+    queryKey: ["backups"],
+    queryFn: () => urbackupServer.getBackupsClients(),
+    refetchInterval: REFETCH_INTERVAL,
```

Comment:
I don't think this needs refetch (clients are not added that often). A shorter cache timeout might be good however

Reply:
I removed the refetch interval. It will use the default now (fetch on window refocus). 

The default cache timeout is 5 minutes. Please let me know if I should change that.